wrapper.py

```diff
@@ -273,22 +276,17 @@ def _load_dataset(f, lazy=False):
         return None
     else:
         if hasattr(f, 'value'):
-            if str(f.value) == 'None':
+            if f.attrs['_value_type'] == 'NoneType':
```

Comment:
Note that I am checking here for the attribute `_value_type`, not for the value itself. `NoneType` is the name of the type of `None`, therefore it is saved like this.